app/views/request/show.html.erb

```diff
@@ -54,17 +54,22 @@
     <div class="request-footer__action-bar-container">
       <div class="request-footer__action-bar__actions">
         <% if @in_pro_area %>
+          <%= render partial: 'request/reply',
+                     locals: { info_request: @info_request,
+                               last_response: @last_response } %>
```

Comment:
I think it would be nicer to render this within `alaveteli_pro/info_requests/after_actions`?

```diff
diff --git a/app/views/alaveteli_pro/info_requests/_after_actions.html.erb b/app/views/alaveteli_pro/info_requests/_after_actions.html.erb
index d9b830634..9230df490 100644
--- a/app/views/alaveteli_pro/info_requests/_after_actions.html.erb
+++ b/app/views/alaveteli_pro/info_requests/_after_actions.html.erb
@@ -1,3 +1,7 @@
+<%= render partial: 'request/reply',
+           locals: { info_request: info_request,
+                     last_response: last_response } %>
+
 <div id="after_actions" class="after-actions">
   <ul class="action-menu after-actions__action-menu">
     <li>
diff --git a/app/views/request/show.html.erb b/app/views/request/show.html.erb
index b184ec6b3..1e7b1fc47 100644
--- a/app/views/request/show.html.erb
+++ b/app/views/request/show.html.erb
@@ -54,9 +54,6 @@
     <div class="request-footer__action-bar-container">
       <div class="request-footer__action-bar__actions">
         <% if @in_pro_area %>
-          <%= render partial: 'request/reply',
-                     locals: { info_request: @info_request,
-                               last_response: @last_response } %>
           <%= render partial: 'alaveteli_pro/info_requests/after_actions',
                      locals: { info_request: @info_request,
                                last_response: @last_response } %>
```